vendor/assets/stylesheets/dvl/core/includes.scss

```diff
@@ -1,16 +1,16 @@
 // #### VARIABLES ####
 
 // Grayscale
-$black: #222;
-$darkestGray: #444;
-$blankStateGray: #666;
-$darkerGray: #777;
-$darkGray: #aaa;
-$gray: #ccc;
+$black: #262626;
```

Comment:
Eventually I want to change these to `$gray1`, `$gray2`, `$gray3`, etc. Seems easier to work with than having to remember the name for each.